scripts/bootstrap.sh

```diff
@@ -104,7 +108,11 @@ env >> /tmp/BOOTSTRAP.PCLUSTER
 pcluster list
 
 # Start the pcluster provisioning, but don't wait for it to complete.
-pcluster create -t hpc hpc-cluster -c ~/.parallelcluster/config --nowait -nr
+pcluster create -t hpc hpc-cluster -c ~/environment/config.ini --nowait -nr -g "{\"NoTearsHPC\": \"${notearshpc_version}\"}"
+if [ $? != 0 ]; then
```

Comment:
What is this for?

Reply:
When launching multiple stacks in the same region, there is a bottlenecked resource on the cluster name. First to pcluster create gets "hpc-cluster", second and subsequent calls to create get "hpc-cluster-${cloud9_environment_id}" (e.g., hpc-cluster-b4875dccb75a459b9c457ae9543eeba0). Identifying which hpc-cluster to use is relatively easy by looking at the Cloud9 URL and matching UUIDs.